docs/Troubleshooting.md

```diff
@@ -215,8 +215,9 @@ This is a known issue an occurs on first upload of Linux data into an OMS worksp
 * Connection to the OMS Service is blocked
 * VM was rebooted
 * OMI package was manually upgraded to a newer version compared to what was installed by OMS Agent package
-* DSC resource logs "class not found" error in omsconfig.log log file
+* DSC resource logs "class not found" error in `omsconfig.log` log file
 * OMS Agent for Linux data is backed up
+* DSC resouce logs "Current configuration does not exist. Execute Start-DscConfiguration command with -Path parameter to specify a configuration file and create a current configuration first." in `omsconfig.log` log file, but no log message exists about `PerformRequiredConfigurationChecks` operations.
```

Comment:
one last thing, it is not dsc resource that logs current configuration does not exist, but the LCM or the engine , I think.

Reply:
you can change to just be 'DSC logs ...."Current..." .."